<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIMenu</class>
 <widget class="QWidget" name="UIMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UIMenu</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="pic_label">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>7</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>47</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学生名字</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="return_pushButton">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>87</y>
     <width>96</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>98</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(5, 157, 213);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="label">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>7</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="check_pushButton">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>120</y>
     <width>96</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>back_label</zorder>
  <zorder>return_pushButton</zorder>
  <zorder>pic_label</zorder>
  <zorder>name_label</zorder>
  <zorder>label</zorder>
  <zorder>check_pushButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
